public/main.css

```diff
@@ -0,0 +1,167 @@
+/**** Header  *****/
+  
+    #header {
+        display: flex;
+        color: orangered;
+        justify-content: center;
+        border: solid;
+        border-color: orangered;
+        border-width: 2px;
+        border-radius: 25px;
+        background-color: deepskyblue;
+        margin-bottom: 1em;
+        font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    }
+
+    a {
+        text-decoration: none;
+        color: orangered;
+    }
+    /********* middle container for sidebar and map ********/
+    .middleContainer {
+        display: flex;
+            }
+    /*********** sidebar **********/
+    nav {
+        display: flex;
+        order: 1;
+        height: 50vh;
+        width: 24vw;
+        color: orangered;
+        justify-content: center;
+        align-content: left;
+        border: solid;
+        border-color: orangered;
+        border-width: 2px;
+        border-radius: 25px;
+        background-color: deepskyblue;
+        margin-right: 1em;
+        font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    }
+    /********* Map section**********/
+    #map {
+        display: flex;
+        order: 2;
+        height: 50vh;
+        width: 74vw;
+        border: solid;
+        border-color: orangered;
+        border-width: 2px;
```

Comment:
You can use the shorthand rule for border like this:
```js
border: 2px solid orangered;
```
and that will set your border type, width, and color in one rule.